cms-adapter/src/Rsbc.Dmf.CaseManagement/Manager/DocumentManager.cs

```diff
@@ -28,6 +28,20 @@ public DocumentManager(DynamicsContext dynamicsContext, ILogger<DocumentManager>
             _configuration = configuration;
         }
 
+        public enum submittalStatusOptionSet
```

Comment:
all public enums, classes, structs, etc should be Pascal case e.g. `SubmittalStatusOptionSet`